contrib/templates/data-engineering/base/template/{{.project_name}}/README.md.tmpl

```diff
@@ -0,0 +1,86 @@
+# {{.project_name}}
+
+The '{{.project_name}}' project was generated by using the data-engineering template.
+
+## Setup
+
+1. Install the Databricks CLI from https://docs.databricks.com/dev-tools/cli/databricks-cli.html
+
+2. Authenticate to your Databricks workspace, if you have not done so already:
+    ```
+    $ databricks auth login
+    ```
+
+3. We recommend the UV package manager to install project dependencies. It's a drop-in replacement for `pip`.
+   See https://docs.astral.sh/uv/getting-started/installation/ for full installation instructions,
+   or run:
+   ```
+   $ pip install uv
+   ```
+
+4. Install all project dependencies:
+   ```
+   $ uv sync
+   ```
+
+   See the "Running unit tests" below for more on testing.
+
+5. Optionally, install developer tools such as the Databricks extension for Visual Studio Code from
+   https://docs.databricks.com/dev-tools/vscode-ext.html. Or the PyCharm plugin from
+   https://www.databricks.com/blog/announcing-pycharm-integration-databricks.
+
+## Adding assets such as pipelines and jobs
+
+By default, the data-engineering template does not include any assets.
+
+1. To add an asset, run the `add-asset` script:
+   ```
+   $ uv run add-asset
+   ```
+
+   or, if you don't use UV, use
+   
+   ```
+   $ export TYPE=etl-pipeline
+   $ databricks bundle init https://github.com/databricks/bundle-examples --template-dir contrib/templates/data-engineering/assets/$TYPE
+   ```
+
+2. Optionally, run all tests on serverless compute after adding an asset:
+   ```
+   $ uv run test
```

Comment:
I get the following error:

```
% uv run test
ImportError while loading conftest '/private/tmp/my_data_project/conftest.py'.
conftest.py:4: in <module>
    import dlt
.venv/lib/python3.13/site-packages/dlt/__init__.py:1: in <module>
    from .api import *
.venv/lib/python3.13/site-packages/dlt/api.py:3: in <module>
    from pyspark.sql.types import StructType
.venv/lib/python3.13/site-packages/pyspark/__init__.py:148: in <module>
    from pyspark.sql import SQLContext, HiveContext, Row  # noqa: F401
.venv/lib/python3.13/site-packages/pyspark/sql/__init__.py:43: in <module>
    from pyspark.sql.context import SQLContext, HiveContext, UDFRegistration, UDTFRegistration
.venv/lib/python3.13/site-packages/pyspark/sql/context.py:39: in <module>
    from pyspark.sql.session import _monkey_patch_RDD, SparkSession
.venv/lib/python3.13/site-packages/pyspark/sql/session.py:48: in <module>
    from pyspark.sql.functions import lit
.venv/lib/python3.13/site-packages/pyspark/sql/functions/__init__.py:20: in <module>
    from pyspark.sql.functions.builtin import *  # noqa: F401,F403
.venv/lib/python3.13/site-packages/pyspark/sql/functions/builtin.py:52: in <module>
    from pyspark.sql.udtf import AnalyzeArgument, AnalyzeResult  # noqa: F401
E   ImportError: cannot import name 'AnalyzeArgument' from 'pyspark.sql.udtf' (/private/tmp/my_data_project/.venv/lib/python3.13/site-packages/pyspark/sql/udtf.py)
```

Reply:
I think this is related to newer Python versions and PySpark.

Locking Python to 3.10 in `pyproject.toml` fixes the issue:
```toml
requires-python = "==3.10.*"
```

Reply:
I see, I was running it with 3.12. May be a DB Connect issue. Let's pin then 🤷

Reply:
Ah. Apparently this is caused by the dependency of `databricks-dlt` on `pyspark`. That makes things really flaky. There is work on a sanity check for this in DB Connect.

Reply:
The workaround should address this. And the updated `databricks-dlt` will fully fix it.